extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -23,7 +23,12 @@ define('ext.wikia.adEngine.lookup.lookupFactory', [
 			response = true;
 			onResponseCallbacks.start();
 
-			adTracker.track(module.name + '/lookup_end', module.getPrices(), 0, 'nodata');
+			if (module.name === 'prebid') {
```

Comment:
only prebid has adapters. The problem with prebid is that we don't want to track the prebid itself, from our perspective it acts like a transparent box. We want to track adapters which are inside this box. In case of other bidders - there is no box, we have direct access to them.